lib/workers/global/initialize.ts

```diff
@@ -35,7 +36,7 @@ export async function globalInitialize(
   config = await initPlatform(config);
   config = await setDirectories(config);
   packageCache.init(config);
-  limits.init(config);
+  limits.setMaxLimit(Limit.Commits, config.prCommitsPerRunLimit);
```

Comment:
Why not have a `init` in `limits` which calls `setMaxLimit` for each type? Do we have multiple init places?

Reply:
Right, here we init `Commits` limit from config, but later we will use more dynamic variant for `PullRequests` and `Branches`